src/stories/containers/TransparencyReport/components/AccountsSnapshot/components/Cards/ReserveCard.tsx

```diff
@@ -308,11 +308,7 @@ const Inflow = styled.div<WithIsLight>(({ isLight }) => ({
   },
 }));
 
-const Outflow = styled(Inflow)({
-  [lightTheme.breakpoints.down('table_834')]: {
-    marginTop: 7,
-  },
-});
+const Outflow = styled(Inflow)({});
```

Comment:
Why don't use the same component, if at the end don't modification was made?
```const Outflow = styled(Inflow)({}) `` 
Why you can don't use Inflow direct?

Reply:
It was done this way to improve the semantic and readability of the code

Reply:
Ok, I got you ponit